Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/ErrorHandlingTransmissionPolicyTest.cs

```diff
@@ -219,7 +285,7 @@ private class TestableErrorHandlingTransmissionPolicy : ErrorHandlingTransmissio
         {
             public TimeSpan BackOffTime { get; set; }
 
-            public override TimeSpan GetBackOffTime(NameValueCollection headers = null)
+            protected override TimeSpan GetBackOffTime(NameValueCollection headers = null)
```

Comment:
Why is this method protected but the plain property above is public?  And why have both?